entrypoint

```diff
@@ -17,6 +17,11 @@ if [[ $# -eq 0 ]]; then
         the source of the \`nextstrain\` command for examples.
     "
 else
-    # Otherwise, exec into whatever command is provided.
-    exec "$@"
+    # Otherwise, exec into whatever command is provided…
+    if [[ "$(id -u):$(id -g)" == 0:0 ]]; then
+        # …switching to nextstrain:nextstrain first if we're root.
+        exec setpriv --reuid nextstrain --regid nextstrain --init-groups "$@"
```

Comment:
I have no idea what `setpriv` does so I went to read the [docs](https://man7.org/linux/man-pages/man1/setpriv.1.html) and saw this bit:

> Setting a uid or gid does not change capabilities, although
           the exec call at the end might change capabilities. This
           means that, if you are root, you probably want to do
           something like:
>
> setpriv --reuid=1000 --regid=1000 --inh-caps=-all

Do this apply here?

Reply:
I'm familiar with the basics of capabilities and a little fuzzier on their specifics, but that recommendation doesn't apply here AFAIK. The Docker daemon which launches the container process is already limiting down capabilities at start, and I think it's fine to keep that limited set inheritable. Note that the former `USER` line in the Dockerfile doesn't do anything related to capabilities. The `setpriv` invocation here is basically ~equivalent to `USER`.